src/main.ts

```diff
@@ -25,15 +27,36 @@ export default class TTSPlugin extends Plugin {
 		}
 
 		await this.loadSettings();
+		if (
+			this.settings.savedVersion !== "0.0.0" && // never installed
+			this.settings.savedVersion !== this.manifest.version // new version
+		) {
+			new TTSPluginUpdateInfo(this.app, this).open();
+		}
 
 		this.addCommand({
 			id: 'start-tts-playback',
 			name: 'Start playback',
 			checkCallback: (checking: boolean) => {
-				const markdownView = this.app.workspace.getActiveViewOfType(MarkdownView);
-				if(!checking && markdownView)
-					this.ttsService.play(markdownView);
-				return !!markdownView;
+				if (!checking) {
+					const markdownView = this.app.workspace.getActiveViewOfType(MarkdownView);
+					if (markdownView) {
+						if (this.ttsService.isSpeaking() && !this.paused) {
+							console.debug("pausing")
+							this.ttsService.pause()
+							this.paused = true
+						}
+						else if (this.paused) {
+							console.debug("resuming")
+							this.ttsService.resume(); this.paused = false
```

Comment:
Please stick to the code style of the rest of the code.
That means semicolons at the end, and one call per line.